src/nvim/shada.c

```diff
@@ -3482,6 +3489,8 @@ static inline char *get_converted_string(const vimconv_T *const sd_conv,
           (tgt) = aetv.vval.v_list; \
         } \
       } while (0)
+// Workaround for silencing linter.
+#define ELSE else
```

Comment:
Or we should just commit without fixing this so that the exception is noted in errors.json.